frontend/src/components/LandingPage/Explanation/Explanation.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Jumbotron } from 'reactstrap'
+//Import components that you create here
+
+function Explanation(){
+    return(
+        //Use components that you import here
+        <Jumbotron>
+        <h1 className="display-3">Explanation</h1>
```

Comment:
What do you think about indenting over the elements that are inside of the jumbotron for readability?

Reply:
Good idea, indenting is imp for sure.